client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMetaFinalEmail.vue

```diff
@@ -0,0 +1,320 @@
+<license>
```

Comment:
I think the non-edit-mode (i.e. when the statement is not assigned to me) still needs some adjustments:

![final_mail](https://github.com/user-attachments/assets/e8e72196-95cb-4c8e-ab73-6d5533d356fa)

Reply:
Thanks for noticing. Done with [3596706](https://github.com/demos-europe/demosplan-core/pull/4407/commits/3596706305d34344349c34bec2d01471f9f285a8)

Reply:
I think that we should also display something below the "Kopie" label in non-edit-mode. I found the text very confusing because also it was not clear to me what "Kopie" meant. Maybe we could rename it to "Kopie an"? And the label hint is not needed when you can't input anything. After looking at what is displayed for other fields on the page when they are disabled and empty, I'd display a message like "Keine E-Mail-Adressen angegeben" or something like that?

Reply:
I removed the hint when disabled and changed the label to "Kopie an" with [ea9eed7](https://github.com/demos-europe/demosplan-core/pull/4407/commits/ea9eed7ffa7235d3725a0f80e3ad493bc7f99a77).

> After looking at what is displayed for other fields on the page when they are disabled and empty, I'd display a message like "Keine E-Mail-Adressen angegeben" or something like that?

Most input fields also have nothing displayed in disabled mode. I suggest to to fix this later overall in a bug ticket.